lib/twitter-ads/resources/analytics.rb

```diff
@@ -199,7 +209,27 @@ def fetch_async_job_data(data_url)
         end
       end
 
-    end
+      def active_entities(account, start_time:, end_time:, **opts)
+        entity_type = name
+        granularity       = opts.fetch(:granularity, nil)
+        start_utc_offset  = opts[:start_utc_offset] || opts[:utc_offset]
+        end_utc_offset    = opts[:end_utc_offset] || opts[:utc_offset]
+
+        if entity_type == 'OrganicTweet'
+          raise "'OrganicTweet' not support with 'active_entities'"
+        end
 
+        params = {
+          entity: ANALYTICS_MAP[entity_type],
+          start_time: TwitterAds::Utils.to_time(start_time, granularity, start_utc_offset),
```

Comment:
I see there's a difference between Python implementation where that one is validating whole hours and letting them pass the hour in, we could also truncate the time to closest hour - let's chat about this offline

Reply:
@jbabichjapan Thanks for asking this. I asked @juanshishido who originally implemented this on Python SDK and he told me that the validation is now done by API side:

```
$ twurl -H ads-api.twitter.com "/5/stats/accounts/18ce54d4x5t/active_entities?start_time=2019-01-01T00:00:00Z&end_time=2019-01-01T00:15:00Z&entity=CAMPAIGN"
{
  "errors": [
    {
      "code": "INVALID_PARAMETER",
      "message": "The end_time (2019-01-01T00:15:00Z) must be expressed in whole hours (0 minutes and 0 seconds)",
      "parameter": "end_time"
    }
  ],
  "request": {
    "params": {
      "account_id": "18ce54d4x5t",
      "entity": "CAMPAIGN",
      "start_time": "2019-01-01T00:00:00Z",
      "end_time": "2019-01-01T00:15:00Z"
    }
  }
}
```

so we don't need to validate this explicitly in the SDK anymore.